### PR TITLE
Improve discussion on HTTP Proxies

### PR DESCRIPTION
**Pull request by pecameron**
This was motivated by:
https://bugzilla.redhat.com/show_bug.cgi?id=1380342

Signed-off-by: Phil Cameron <pcameron@redhat.com>

### CODE REVIEW

**Comment by pecameron** [comment]
@knobunc PTAL

File: install_config/http_proxies.adoc [hunk 1]
```diff
@@ -20,62 +20,86 @@ configuration or JSON files.
...
-<1> Supports host names and CIDRs. Must include the SDN and service IP ranges `10.1.0.0/16,172.30.0.0/16` by default.
-====
-+
+
+[[configuring-no-proxy]]
+== Configuring NO_PROXY
+The `*NO_PROXY*` environment variable lists all of the {product-title}
+components and all IP addresses that are managed by {product-title}.
+
 `*NO_PROXY*` accepts a comma-separated list of hosts:
```
Comment (knobunc) [inline install_config/http_proxies.adoc]: hosts and CIDRs and IPs?

**Review by knobunc — CHANGES_REQUESTED** [review]

File: install_config/http_proxies.adoc (hunk 1 above)
Comment (pecameron) [inline install_config/http_proxies.adoc]: Yes. I made the change.

**Comment by pecameron** [comment]
@knobunc PTAL

**Comment by pecameron** [comment]
@knobunc I made the change you suggested. Is there more to do here? PTAL

**Review by knobunc — APPROVED** [review]
LGTM.  Over to you @openshift/team-documentation

**Comment by knobunc** [comment]
@pecameron presumably this applies to all releases they want to put it in?  (3.2 and up)

**Comment by vikram-redhat** [comment]
@ahardin-rh - PTAL. Also @knobunc - applies to 3.2 and above?

**Comment by ahardin-rh** [comment]
Overall, LGTM! I will apply some minor follow-up edits in a separate PR. Thanks!

**Comment by knobunc** [comment]
@vikram-redhat @ahardin-rh: I believe it applies to all releases ( @pecameron please confirm).  But you needn't apply it to all of those versions if you don't want to.

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/http_proxies.adoc#install-config-http-proxies[Working with HTTP Proxies]
|Added clarifying details about HTTP proxies.
%

**Pull request merged**